<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RumidiaPlayer</class>
 <widget class="QMainWindow" name="RumidiaPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RumidiaPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No File Loaded</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadFileBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="soundFontBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Sound Font</string>
    </property>
    <widget class="QLabel" name="curSFLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>180</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>No SoundFont Loaded</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadSFBtn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Load...</string>
     </property>
    </widget>
    <widget class="QLabel" name="SFNameLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>45</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>SoundFont Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="SFInsLoadedLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>SoundFont Loaded:</string>
     </property>
    </widget>
   </widget>
   <widget class="SPSlider" name="progressSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="playBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="controlBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Control Panel</string>
    </property>
    <widget class="QSlider" name="tempoSlider">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>22</width>
       <height>101</height>
      </rect>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="tempoLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tempo</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="NOFXCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Reverb&amp;Chorus</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="SPSlider" name="volSlider">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SPSlider</class>
   <extends>QSlider</extends>
   <header>spslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
